source/exhumed/src/gun.cpp

```diff
@@ -1036,8 +1037,37 @@ void DrawWeapons(int smooth)
     }
 
     if (nWeapon == 3 && var_34 == 1) {
+        // If the weapon is the flamer, draw the pilot light that sits at the muzzle.
         seq_DrawPilotLightSeq(xOffset, yOffset);
     }
```

Comment:
Don't think this check for `windowxy2.y` being `0` was necessary but did it anyway to de-risk division by zero. If a division by zero situation were to happen, this block skips and the gun sequence ends up being drawn with the same x offset it did before this change (160, see `sequence.cpp:492).